components/molecule/selectPopover/src/index.js

```diff
@@ -0,0 +1,83 @@
+import React, {useState, useEffect, useRef, useCallback} from 'react'
+import Button from '@s-ui/react-atom-button'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+
+const BASE_CLASS = `sui-MoleculeSelectPopover`
+
+export default function MoleculeSelectPopover({
+  acceptButtonText,
+  cancelButtonText,
+  children,
+  iconArrowDown: IconArrowDown,
+  isSelected = false,
+  onAccept,
+  onCancel,
+  selectText
+}) {
+  const [isOpen, setIsOpen] = useState(false)
+
+  const popoverRef = useRef()
+
+  const selectClassName = cx(`${BASE_CLASS}-select`, {
+    [`is-open`]: isOpen,
```

Comment:
It's weird that linter is not complaining about using template string when is not needed.